src/components/CarouselLinks/CarouselLinks.tsx

```diff
@@ -14,28 +14,26 @@ export interface CarouselLinksProps extends CarouselLinksContent, UniBlockProps
 const MAX_LINKS_COUNT = 12;
 
 export const CarouselLinks = JSX<CarouselLinksProps>(
-  ({ additionalDescription, block, description, links, title, className = '', ...rest }) => {
-    return (
-      <BlockWrapper
-        className={`bg-primary-main flex w-full p-10 overflow-hidden justify-between font-sans ${className}`}
-        {...rest}
-      >
-        <div className="flex flex-col max-w-96 pr-9 text-white text-xl-light">
-          {title ? <Heading title={title} as="h1" headingType="h0" className="pb-0.5" /> : null}
-          {description ? (
-            <Description description={description} className="text-xl-light pt-5" />
-          ) : null}
-          {additionalDescription ? (
-            <Description description={additionalDescription} className="pt-6" />
-          ) : null}
-        </div>
-        {links?.length ? renderLinks(links, block) : null}
-      </BlockWrapper>
-    );
-  },
+  ({ additionalDescription, description, links, title, className = '', ...rest }) => (
+    <BlockWrapper
+      className={`bg-primary-main flex w-full p-10 overflow-hidden justify-between font-sans ${className}`}
+      {...rest}
+    >
+      <div className="flex flex-col max-w-96 pr-9 text-white text-xl-light">
+        {title ? <Heading title={title} as="h1" headingType="h0" className="pb-0.5" /> : null}
+        {description ? (
+          <Description description={description} className="text-xl-light pt-5" />
+        ) : null}
```

Comment:
```suggestion
          <Headline
            title={title}
            as="h1"
            description={description}
            align="left"
            className="p-0"
            bgColorHeadline="secondary"
          />
```
Можешь в рамках рефакторинга поправить этот момент? Обсудили с дизайнерами, что тут подойдет title с размером 50px, что соответствует Headline XL

Reply:
Не согласен. не стоит всё складывать в один МР, лучше создать атомарный МР который быстро проверить и влить так же CarouselLink переименовать в CarouselLinkCard